ops/docker/ci-builder/Dockerfile

```diff
@@ -11,12 +11,15 @@ RUN apt-get update && \
     chmod +x ./rustup.sh && \
     ./rustup.sh -y
 
+# move the foundryrc file to the foundry dir
+COPY .foundryrc /opt/foundry
```

Comment:
This doesn't work, just my best guess at it.

Reply:
Looks like we could pass in the version using `docker build --build-arg foundryrc=$(cat .foundryrc)`.

Reply:
Probably worth setting this cmd as a `Makefile` target then so we don't need to think about it